### DISCUSSION TITLE
Supporting 256 and RGB colors

### DISCUSSION THREAD

**Discussion by AlbatorLaho** [tui-cs/Terminal.Gui#1579] [Q&A]
Is there a way to use 256 or RGB colors? I know that NCurses at least supports 256 (I'm not sure about RGB) and I think the Windows 10 Console supports 256 and RGB... so is this currently possible in Terminal.Gui? or will it be in some future update?

**Comment by BDisp** [comment]
Currently isn't supported but I hope it'll be implemented some day.
